package.json

```diff
@@ -43,6 +43,8 @@
     "setup": "npm run module && npm run virtualenv && npm install && npm run config",
     "watch" : "grunt watch",
     "build": "grunt",
+    "lint": "grunt eslint",
```

Comment:
@zoepage ah, if we document eslint anywhere in CONTRIBUTING.md, it might be good to add these commands as alternatives.